### PR TITLE
Update design system to v70

### PR DESCRIPTION
**Pull request by matthew-robinson-ons**
# What and why?
This Pr updated the design system to the latest available version at the time. 
Strong tags in headings are now used to highlight so these have been removed from the code to prevent any unintended Ui changes. 
# How to test?
Run the app and check functionality and UI, Pagination headings should display the same as before. 
# Jira
https://jira.ons.gov.uk/browse/RAS-1094

### CODE REVIEW

**Review by arroyoAle — APPROVED** [review]
tests pass, changes look good, nothing seems to have changed in the UI

**Review by LJBabbage — CHANGES_REQUESTED** [review]
Looks like you might have missed this change https://github.com/ONSdigital/design-system/pull/3047. Not sure about those strong tags, I can see what they have done, but doesn't that stop them being bold?

**Comment by matthew-robinson-ons** [comment]
> Looks like you might have missed this change [ONSdigital/design-system#3047](https://github.com/ONSdigital/design-system/pull/3047). Not sure about those strong tags, I can see what they have done, but doesn't that stop them being bold?

Will take a look at the link, with the strong tags, removing them had no visual change, i believe H3 is already bold so the strong tags were redundant anyway.

File: response_operations_ui/templates/account/my-account.html [hunk 1]
```diff
@@ -12,13 +12,14 @@
...
     <h1 id="respondent-name">Account details</h1>
     <dl class="ons-metadata ons-metadata__list ons-grid ons-grid--gutterless ons-u-cf ons-u-mb-l" title="">
-        <dt class="ons-metadata__term ons-grid__col ons-col-4@m">Email address:</dt>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m">{{ user.email }}</dd>
-        <dt class="ons-metadata__term ons-grid__col ons-col-4@m">Name:</dt>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m">{{ user.name }}</dd>
-        <dt class="ons-metadata__term ons-grid__col ons-col-4@m">Password:</dt>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m">●●●●●●●●</dd>
-        <dd class="ons-metadata__value ons-grid__col ons-col-8@m ons-u-ml-no ons-u-mb-xs ons-u-fs-s">(Last changed on {{ user.password_last_changed }})</dd>
+        <dt class="ons-metadata__term ons-grid__col ons-col-4@m"><strong>Email address:</strong></dt>
```
Comment (matthew-robinson-ons) [inline response_operations_ui/templates/account/my-account.html:15]: Off the back of the fronstage PR have made these changes as well. This keeps the same visual format as before it seems ons-metadata__term was adding bold text styling and is no longer doing that.

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**